docs/customization/template.rst

```diff
@@ -0,0 +1,91 @@
+Customizing Templates
+=====================
+
+.. note::
+
+    There are two kinds of templates in Sylius. **Shop** and **Admin** ones, plus you can create your own to satisfy your needs.
+
+Why would you customize a template?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The most important case for modifying the existing templates is of course **integrating your own layout of the system**.
+Sometimes even if you have decided to stay with the default layout provided by Sylius you need to **slightly modify it to meet your
+business requirements**.
+You may just need to **add your logo anywhere**.
+
+How to customize templates?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. note::
+
+    How do you know which template you should be overriding?
+    Go to the page that you are going to modify, at the bottom in the Symfony toolbar click on the route name
+    which will redirect you to the profiler, where in the Request Attributes section
+    under ``_sylius [ template => ...]`` you can check the route to the current template.
```

Comment:
This is pretty long sentence 😄  I would split it into two, to improve readability.